.github/workflows/snap.yml

```diff
@@ -34,6 +34,37 @@ jobs:
         name: snap
         path: ${{ steps.snapcraft.outputs.snap }}
 
+  test-snap:
+    needs: build-snap
+    runs-on: ubuntu-latest
+    steps:
+    - id: checkout
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - id: install-deps
+      run: |
+        sudo apt update
+        sudo apt install -y snapd gcab
+    - uses: actions/download-artifact@v4
+      id: download
+      with:
+        name: snap
+    - id: install-snap
+      run: |
+        sudo snap install --dangerous ${{ needs.build-snap.outputs.snap_name }}
+        sudo mkdir -p /var/snap/fwupd/common/var/etc/fwupd
+        sed "s,^DisabledPlugins=.*,DisabledPlugins=," /snap/fwupd/current/etc/fwupd/fwupd.conf | sudo tee /var/snap/fwupd/common/var/etc/fwupd/fwupd.conf
```

Comment:
i wondered if we should have a fwupdtool/fwupdmgr command to do this; it's something we seem to do a lot

Reply:
We can probably do it with `fwupdmgr modify-config DisabledPlugins`.

Reply:
Hmm; modify-config is broken...  Look what a few runs are doing:

```
[fwupd]
DisabledPlugins=test;test_ble
OnlyTrusted=true
AllowEmulation=false
DisabledPlugins=
OnlyTrusted=true
AllowEmulation=false
DisabledPlugins=test;test_ble
OnlyTrusted=true
AllowEmulation=false
DisabledPlugins=test_ble
OnlyTrusted=true
AllowEmulation=false
```

Reply:
Given that bug I think should leave as it for introducing the test and clean up later.

Reply:
Eeek, can you file it and I'll fix it on Monday please.